test/jdk/java/nio/file/Files/CreateDirectories.java

```diff
@@ -120,5 +120,13 @@ public void testCreateDirectories() throws IOException {
         Path root = Path.of("/");
         Files.createDirectories(root);
         Files.createDirectories(root.toAbsolutePath());
+
+        // the returned path should not be absolute
+        Path temp = Path.of(".temp/temp.abc/temp.def");
+        Files.deleteIfExists(temp);
+        Files.deleteIfExists(temp.getParent());
+        temp.toFile().deleteOnExit();
+        Path a = Files.createDirectories(temp);
+        assertTrue(a == temp, a + " != " + temp);
```

Comment:
The spec is that Files.createDirectories returns the dir parameter so I think we can change L94 to check that Files.createDirectories(tmpdir) returns tmpdir.

For the case where the dir is a relative path then probably no need for the Files.delete + deleteOnExit as they will be created in the test work directory where it's useful to leave behind crumbs.